setup.py

```diff
@@ -25,5 +32,5 @@
         "License :: OSI Approved :: MIT License",
         "Operating System :: OS Independent",
     ],
-    python_requires='>=3.6',
-)
\ No newline at end of file
+    python_requires='>=3.5',
```

Comment:
This package works on Python 3.5 as well. It probably works on even older versions... but I've just put Python 3.5 here anyways.